docs/configuration/secrets.md

```diff
@@ -7,61 +7,61 @@ nav_order: 8
 
 # Secrets
 
-Configuration of Authelia requires some secrets and passwords.
-Even if they can be set in the configuration file, the recommended
-way to set secrets is to use environment variables as described
-below.
+Configuration of Authelia requires some secrets and passwords. Even if they can be set in the configuration file or 
+standard environment variables, the recommended way to set secrets is to use environment variables as described below.
 
 ## Environment variables
 
-A secret can be configured using an environment variable with the
-prefix AUTHELIA_ followed by the path of the option capitalized
-and with dots replaced by underscores followed by the suffix _FILE.
+A secret value can be loaded by Authelia when the configuration key ends with one of the following words: `key`, 
+`secret`, `password`, or `token`. 
 
-The contents of the environment variable must be a path to a file
-containing the secret data. This file must be readable by the
-user the Authelia daemon is running as.
+If you take the expected environment variable for the configuration option with the `_FILE` suffix at the end. As long
+as the value of this environment variable is the path of a file, Authelia will load the contents, trim any newlines at
```

Comment:
> As long as the value of this environment variable

You should I either say "the value of this environment variable should be the path of a file" or describe what happens if it's not a path of a file.